includes/interwiki/Interwiki.php

```diff
@@ -295,6 +307,8 @@ protected static function getAllPrefixesDB( $local ) {
 		foreach ( $res as $row ) {
 			$retval[] = (array)$row;
 		}
+
+		$wgMemc->set( $key, $retval, $wgInterwikiExpiry ); // Wikia change
```

Comment:
Oh. Right! This code was collapsed and I didn't noticed. Then it is OK the way it is.